Frameworks/CoreGraphics/CGGraphicBufferImage.mm

```diff
@@ -174,6 +179,7 @@
     int stride;
     _imageData = _nativeTextureLocking->LockWritableBitmapTexture(_nativeTexture, &stride);
     _bytesPerPixel = 4;
+    _bitsPerComponent = 8;
```

Comment:
I verified that indeed there is no reason why bitsPerComponent nor bytesPerPixel needs to be set here or in BytesPerRow where it is also overridden.